packages/endpoints-organizations-registrar/src/entities/registrar-consents/repos/registrar-consent-repo-extension.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2023 Velocity Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { ObjectId } = require('mongodb');
+const { nanoid } = require('nanoid');
+
+const registrarConsentRepoExtension = (parent) => ({
+  registerConsent: (
```

Comment:
please refactor other inserts to this collection

Reply:
hows this [e17d20a](https://github.com/velocitycareerlabs/velocitycore/pull/34/commits/e17d20a209a4c8c14c0872774df05662e21aa1d0)